### ISSUE TITLE
Incorrect capitalized title and side menu link for radiogroup role

### ISSUE THREAD

**Issue by victoriacheng15** [mdn/content#19065]
### MDN URL

https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/radiogroup_role

### What specific section or headline is this issue about?

incorrect capitalized title and side menu link

### What information was incorrect, unhelpful, or incomplete?

All titles are started with `ARIA:`  in uppercase while this radiogroup role title started with `aria:` in lowercase

### What did you expect to see?

`ARIA` should be in uppercase 

### Do you have any supporting links, references, or citations?

This is from the side menu,
![image](https://user-images.githubusercontent.com/35031228/182436031-120b7403-66f7-4b7b-98ca-f5fec18aab95.png)

This is the title from the radiogroup role page,
![image](https://user-images.githubusercontent.com/35031228/182436119-8aaf068f-01c2-4eea-8f21-a4d240d4c42b.png)

Page title from another page
![image](https://user-images.githubusercontent.com/35031228/182437706-e4c9dd5b-64b0-4065-9f5a-6b7b20ba8631.png)

### Do you have anything more you want to share?

n/a

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/radiogroup_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/radiogroup_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/radiogroup_role/index.md
* Last commit: https://github.com/mdn/content/commit/3d0853bb6e643ffb85d2bad5ef5630de0c03a80d
* Document last modified: 2022-06-17T02:58:40.000Z

</details>

**Comment by sideshowbarker** [comment]
Thanks much for catching this and reporting it — I’ve opened #19088 with a fix.

After that fix is merged, this issue will automatically be marked as resolved, and closed.
